test/builder-api/src/origins.js

```diff
@@ -33,7 +33,10 @@ describe('Origin API', function () {
       request.get('/depot/origins/neurosis')
         .expect(200)
         .end(function (err, res) {
-          expect(res.body).to.deep.equal(global.originNeurosis);
+	      expect(res.body.name).to.equal(global.originNeurosis.name);
```

Comment:
I don't even want to type this, but...alas - whitespace

Reply:
oh dang!

Reply:
Actually... there isn't extra whitespace there in the code i'm currently looking at, could this be a GH rendering thing?